src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,362 @@
+
+### Manual injection [#manual]
+If you have  any concerns about the automatic injection, you can inject the sidecar manually directly by modifying the manifests of the workloads scheduled that are going to be scheduled on Fargate nodes. Please note that adding the sidecar into deployments scheduled into EC2 nodes may lead into incorrect or duplicate data, especially if those nodes are already being monitored with the DaemonSet.
+
+The following objects are required for the sidecar to successfully report data:
+
+* The `ClusterRole` providing the permission needed by the `nri-kubernetes integration`
+* A `ClusterRoleBinding` linking the `ClusterRole` and the service account of the pod
+* The secret storing the New Relic `licenseKey` in each Fargate namespace 
+* The sidecar container in the spec template of the monitored workload
+
+#### Manual injection installation [#manual-injection-install]
+
+<Callout variant="tip">
+These manual setup steps are for a generic installation. Before completing these, take a look at the [Configuration](#config-manual) section below to see if you want to modify any aspects of the automatic injection.
+</Callout>
+
+Complete the following for manual injection:
+
+1. If `ClusterRole` does not exist, create it and grant the permissions required to hit the metrics endpoints. This only needs to be done once, even for monitoring multiple applications in the same cluster.
+
+    <CollapserGroup>
+      <Collapser
+        id="cluster-role"
+        title="clusterRole.yaml"
+      >
+      You can use this snippet as it appears below, without any changes:
+
+      ```      
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRole
+      metadata:
+        labels:
+          app: newrelic-infrastructure
+        name: newrelic-newrelic-infrastructure-infra-agent
+      rules:
+      - apiGroups:
+        - ""
+        resources:
+        - nodes
+        - nodes/metrics
+        - nodes/stats
+        - nodes/proxy
+        - pods
+        - services
+        verbs:
+        - get
+        - list
+      - nonResourceURLs:
+        - /metrics
+        verbs:
+        - get
+      ```
+      </Collapser>
+    </CollapserGroup>
+
+2. For each workload you want to monitor, add an additional sidecar container for the `newrelic/infrastructure-k8s` image. Here is an example of an injected sidecar. 
+
+    <CollapserGroup>
+      <Collapser
+        id="container-to-inject"
+        title="containerToInject.yaml"
+      >
+      [PLEASE INSERT INSTRUCTIONS ABOUT HOW TO USE THIS SNIPPET]
```

Comment:
"Take the container of the following snippet and inject it in the workload you want to monitor. Note that the volumes can be defined as `emptyDir: {}`.
Note that in the special case of a KSM deployment you need to remove as well the `DISABLE_KUBE_STATE_METRICS` environment variable and increase the resources requests and limits".

Reply:
@paologallinaharbur, I inserted your new content. Please double-check it.